src/hermes/commands/deposit/invenio.py

```diff
@@ -4,17 +4,21 @@
 
 # SPDX-FileContributor: David Pape
 
-from datetime import date
+import json
+import logging
+from datetime import date, datetime
 
-import requests
+import click
 
 from hermes.model.context import CodeMetaContext
 from hermes.model.path import ContextPath
 
+from .error import DepositionUnauthorizedError
```

Comment:
**nitpick:** I'm not a big fan of relative imports as they might end up doing weird stuff.
In addition, I'm not quite cerain whether we really want to import the class and not only the module - I'm a big fan of namespaces.
However, here this might be okay for reasons, but we should discuss whether some coding guidelines should be applied for similar cases.

```suggestion
from hermes.commands.deposit.error import DepositionUnauthorizedError
```